src/instructlab/sdg/generate_data.py

```diff
@@ -105,27 +105,40 @@ def _convert_to_messages(sample):
     return sample
 
 
-def _gen_train_data(logger, output_datasets, output_file_train):
+def _gen_train_data(
+    logger, machine_instruction_data, output_file_train, output_file_messages
+):
     train_data = []
-    for output_dataset in output_datasets:
-        for synth_example in output_dataset:
-            logger.debug(synth_example)
-            user = _get_question(logger, synth_example)
-            if len(synth_example.get("context", "")) > 0:
-                user += "\n" + synth_example["context"]
-            train_data.append(
-                {
-                    "system": _SYS_PROMPT,
-                    "user": _unescape(user),
-                    "assistant": _unescape(_get_response(logger, synth_example)),
-                }
-            )
+    messages_data = []
+    for synth_example in machine_instruction_data:
```

Comment:
It looks like this dropped the extra `for` loop that was added around this

Reply:
fixed this and re introduced the for loop